src/Microsoft.DotNet.Build.Tasks/Targets/toolruntime.targets

```diff
@@ -14,8 +14,10 @@
     <ToolHost Condition="'$(OS)' != 'Windows_NT'">corerun</ToolHost>
 
     <ToolTargetFramework Condition="'$(ToolTargetFramework)' == ''">DNXCore,Version=v5.0</ToolTargetFramework>
+    <ToolOS Condition="'$(OS)' == 'Windows_NT'">win7</ToolOS>
+    <ToolOS Condition="'$(OS)' != 'Windows_NT'">ubuntu.14.04</ToolOS>
```

Comment:
Ok, I'm adding osx and I'll start looking into generating the project.json on the fly. I'll push my changes shortly.